src/lib/components/dashboard/credentials/CredentialCard.svelte

```diff
@@ -1,69 +1,100 @@
 <script lang="ts">
+    import { writable } from 'svelte/store';
     import CopyIcon from "../../basic/copyIcon.svelte";
     import { CredentialDetails } from "../dtos";
     import { createEventDispatcher } from "svelte";
     import { fly } from "svelte/transition";
     import { fetchCredentialById } from "../apis";
     import browser from "webextension-polyfill";
+  import More from '../../basic/more.svelte';
+  import Locked from '../../basic/locked.svelte';
+  import Eye from '../../basic/eye.svelte';
+  import Unlocked from '../../basic/unlocked.svelte';
+  import SensitiveEye from '../../basic/sensitiveEye.svelte';
+  import SensitiveEye2 from '../../basic/sensitiveEyeBlue.svelte';
     const dispatch = createEventDispatcher();
 
     export let credential;
+    export let visibility = writable(false)
```

Comment:
why is visiblity a store var?

Reply:
export is usually used when its a prop from parent component, refer how credentialList calls credentialCard.